core/src/stages/generate_place_pose.cpp

```diff
@@ -53,6 +53,7 @@ GeneratePlacePose::GeneratePlacePose(const std::string& name)
 	auto& p = properties();
 	p.declare<std::string>("object");
 	p.declare<geometry_msgs::PoseStamped>("ik_frame");
+	p.declare<bool>("augment_rotations");
```

Comment:
Please add a property description and the default of `true` to maintain the previous behavior.